notebooks/245-oneformer-segmentation/245-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>&gt;&gt;Here we use the <code>from_pretrained</code> method of <code>OneFormerForUniversalSegmentation</code> to load the Swin-L model trained on COCO dataset. </p><p>not fully clear, Swin-L is backbone model for OneFormer, it can be used as independent model. Could you please rewrite?</p><p>Also please add links on COCO and HuggingFace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>